workflows/assessment.sw.json

```
@@ -4,21 +4,42 @@
   "specVersion": "0.8",
   "name": "Assessment Workflow",
   "description": "Simple workflow to simulate assessment",
-  "annotations": ["workflow-type/assessment"],
+  "annotations": [
+    "workflow-type/assessment"
+  ],
   "dataInputSchema": "schemas/assessment__main-schema.json",
   "start": "Inject options",
   "states": [
     {
       "name": "Inject options",
       "type": "inject",
       "data": {
-        "workflowOptions": [
-          { "id": "yamlgreet", "name": "Greeting workflow" },
-          { "id": "wait-or-error", "name": "Wait or Error" },
-          { "id": "hello_world", "name": "Hello World Workflow" }
-        ]
+        "workflowOptions": {
+          "newOptions": [
+            {
+              "id": "yamlgreet",
+              "name": "Greeting workflow"
+            },
+            {
+              "id": "wait-or-error",
+              "name": "Wait or Error"
+            },
+            {
+              "id": "hello_world",
+              "name": "Hello World Workflow"
+            }
+          ],
+          "otherOptions": [],
+          "currentVersion": {
+            "id": "yamlgreet",
+            "name": "Dummy infra workflow option"
```

I think we can use the real name of the workflow here
```suggestion
            "name": "Greeting workflow"
```